webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/DeploymentService.scala

```diff
@@ -925,17 +934,29 @@ trait PromiseGeneration_buildRuleVals extends PromiseGenerationService {
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 trait PromiseGeneration_buildNodeConfigurations extends PromiseGenerationService {
+
+  def isPostgresqlLocal: Boolean
+
+  override def getFilteredTechnique(): Map[NodeId, List[TechniqueName]] = {
+    if(isPostgresqlLocal) {
```

Comment:
shouldn't it be the contrary?

Reply:
no, we filter out technique `rudder-service-postgresql` when postgres is remot. If it's local, we don't filter out anything